admin/admin-classic-editor.php

```diff
@@ -110,10 +110,11 @@ public function post_language() {
 			$this->model->get_languages_list(),
 			-1,
 			array(
-				'name'     => $id,
-				'class'    => 'post_lang_choice tags-input',
-				'selected' => $lang ? $lang->slug : '',
-				'flag'     => true,
+				'name'       => $id,
+				'class'      => 'post_lang_choice tags-input',
+				'selected'   => $lang ? $lang->slug : '',
+				'flag'       => true,
+				'value'      => 'slug',
```

Comment:
You can rever this change. The default value for `value` is already `slug`.
See: https://github.com/polylang/polylang/blob/3.1/include/walker-dropdown.php#L103